src/content/docs/contributing/website_markdown.md

```diff
@@ -131,21 +131,23 @@ Many people have speculated that if we knew exactly why the bowl of petunias had
 
 ## GitHub style admonitions
 
-For `README.md` files, you can also use the [GitHub style admonitions/alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts):
+For `README.md` files, you can also use the [GitHub style admonitions/alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).
+This _only_ works with markdown files fetched from pipeline repositories, so don't use it in markdown files in the website repo.
 
 ```md
 > [!NOTE]
-> A magician wandered along the beach, but no one needed him.
+> Useful information that users should know, even when skimming content.
```

Comment:
![](https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExczJkZWl2cnI1dGNvejNnOHg0dHgxOGgybWd2ZDJjMzdpbG56N3ZtaCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/qSQ3Z5gIrlaZa/giphy.gif)

Reply:
hah, it's really true - no one needed him! 🧙🏻